sql/wsrep_sst.cc

```diff
@@ -1127,6 +1128,12 @@ int wsrep_remove_sst_user(bool initialize_thread) {
   int err = 0;
   MYSQL_SESSION session = NULL;
 
+  // Skip the attempt to  mysql.pxc.sst.user in case the server was started with
+  // --skip-grant-tables option. It would fail enyway with error.
+  // This will prevent writing out error to error log.
+  if (skip_grant_tables()) {
```

Comment:
This won't fix the problem completely, as there are a few other edge cases when we can't execute this statement during shutdown, I ended up removing these startup drop statements completely, as some of them can't be fixed, and we start the server with `DROP IF EXISTS` anyway.

We can keep this here, but I'll completely remove this method when I finally have a chance to finish up my bootstrap script cleanup PR.

Reply:
As discussed on slack, feel free to remove it in the future. This just resurrects the MTR test that was extended by me to test fixed functionality.